docs/HOL_Fault-injection-for-Azure-Container-Services(ACS)/README.md

```diff
@@ -0,0 +1,168 @@
+HOL - Fault injection for Azure Container Services (ACS)
+========================================================
+
+
+## Pre-requisites:
+
+- HOL - Deploying Parts Unlimited MRP on DC/OS
+
+- Access to VSTS to run a load test
+
+
+
+## Tasks Overview:
+**1. Set up load test** This will walk your through creating and configuring a load test for the next task using VSTS.
+
+**2. Inject faults into ACS environment using UI** In this task you will simulate faults of different parts of the system by killing its instances in Marathon service installed on DC/OS.
+
+**3. Inject faults into ACS environment using API** In this task you will learn how to achieve exactly the same outcome as in the second task but using Marathon's API.
+
+**4. Analyze results** In this task you will collect all of the information related to the load test and will draw a conclusion about the stability of the environment.
+
+
+
+## HOL:
+
+### Task 1: Set up load test
+
+**Step 1.** Open VSTS for your project, then navigate to "Load Test" page.
+
+![](media/1.png)
+
+
+**Step 2.** Click on "New" and select "URL based test".
+
+![](media/2.png)
+
+
+**Step 3.** Give this load test a name, rename URL name into something more meaningful and enter the MRP solution's URL (`{YourDnsName}/mrp`) into the URL field.
+
+![](media/3.png)
+
+
+**Step 4.** Click on "Add URL", give this URL a meaningful name and enter `{YourDnsName}/quotes?name` as a URL. This query will call your API, which then queries database.
+
+![](media/4.png)
+
+
+**Step 5.** Click on "Settings" and enter the following settings:
+* Run duration (minutes): `20`
+* Load pattern: `Constant`
+* Max v-users: `600`
+* Warmup duration (seconds): `60`
+* Browser mix: leave default value.
+* Load location: Closest location to the datacenter where MRP solution is hosted.
+
```

Comment:
Please add short descriptions about the settings.

e.g. What is warm-up duration?